intro.js

```diff
@@ -977,20 +977,21 @@
     nextTooltipButton.innerHTML = this._options.nextLabel;
 
     if (this._currentStep == 0 && this._introItems.length > 1) {
-      prevTooltipButton.className = 'introjs-button introjs-prevbutton introjs-disabled hidden';
+      // prevTooltipButton.className = 'introjs-button introjs-prevbutton introjs-disabled hidden';
+      prevTooltipButton.className = 'introjs-button introjs-prevbutton float--left';
```

Comment:
Anything related to intro.js should go in its respective file, here it should be in `_intro.scss`. @chinchang 
Also, why do we need to change it here, it's configurable. Check `FeatureWalkthrough.js`.
And about `Do not show me this again` link - https://www.dropbox.com/s/9l83j1pivzk3qwo/Screenshot%202015-11-27%2012.55.10.png?dl=0 ?